EXPANDING ACNE DATASET

For this project, I'm working on expanding a publicly available dataset based off of a paper that compared normal vs. acne skin using single-cell RNA sequencing (scRNAseq). scRNAseq is an effective tool for analyzing genetic and molecular profiles of individual cells. This allows us to study changes in specific cell types based on the types of disease lesions. We can know which genes are differentially expressed in each cell type and each condition. In this case, we can know how gene expression differs between cell types in normal vs. acne skin lesions. It's a powerful way for elucidating unknown pathogeneses and identifying potential targets for therapeutics.

The publicly available data can be found here: https://github.com/modlab246/scRNAacne.

The paper I'm referencing is cited here: Tran H. Do et al. ,TREM2 macrophages induced by human lipids drive inflammation in acne lesions.Sci. Immunol.7,eabo2787(2022).DOI:10.1126/sciimmunol.abo2787.

Do, Tran H. et al. already did quality control and integrated the data, but my goal is to expand a subset of the data. I'm choosing to expand the lymphocyte dataset, which can be downloaded here: https://drive.google.com/drive/folders/1xy32aQAXP0oXH9X7wfqgOTfsFBz1uK-Q.

Originally, the authors named 5 cell clusters in this dataset, however, I plan to figure out if we can expand it into more cell types so that the data is more specific for generating hypotheses. This will allow researchers to more specifically narrow down where a gene of interest is being expressed. After expanding the dataset, I plan to analyze the single cell data in through different graphical representations.
